frontend-opal/components/Header2/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {
+  Button,
+  View,
+  TextInput,
+  Image,
+  Text,
+} from 'react-native';
+import styles from './styles.js';
+
+const Header = () => {
+  const narTitles = [
+    { key: 'PORTFOLI' },
+    { key: 'WATCHLIST' },
+  ];
+  const renderItems = (titles) => (
+    titles.map((title) => (
+      <View style={styles.narTitle} key={title.key}>
+        <Button title={title.key} color="#e17055" />
+      </View>
+
+    ))
+  );
+  return (
+    <View>
+      <View style={styles.search}>
+        <Image style={styles.listIcon} source={require('../../assets/list-icon.png')} />
+        <Text style={styles.input}>Investments</Text>
+        <Image style={styles.searchButton} source={require('../../assets/search-button.png')} />
+      </View>
+
+      <View style={styles.navBars}>
+        {renderItems(narTitles)}
```

Comment:
@martinupset this is generally a bad practice, you call a function to create React components, that is exactly what a functional component is, so why not make it one? But even better, use nativebase

Reply:
ok